.gitattributes

```diff
@@ -0,0 +1,13 @@
+.* export-ignore
+*.md export-ignore
+tests export-ignore
+docs export-ignore
+examples export-ignore
+mongo-orchestration export-ignore
```

Comment:
[This directory](https://github.com/mongodb/mongo-php-library/tree/master/mongo-orchestration) hasn't been updated in over two years and I think it's obsolete since stopped using Travis CI and migrated to GitHub Actions (including the mongo-orchestration action in [drivers-evergreen-tools](https://github.com/mongodb-labs/drivers-evergreen-tools/)). I also found no references to this directory in the project.

@alcaeus: Do you concur that we should remove it entirely?

Reply:
Opened [PHPLIB-1058](https://jira.mongodb.org/browse/PHPLIB-1058) to track this.